internal/agent/install.go

```diff
@@ -53,16 +53,21 @@ func displayInfo(agentConfig *Config) {
 	}
 }
 
-func ManualInstall(c, device string, reboot, poweroff, strictValidations bool) error {
+func ManualInstall(c, sourceImg, device string, reboot, poweroff, strictValidations bool) error {
```

Comment:
I would also extend it to somethign like `sourceImgURL` to make it clearer to consume

Reply:
done